tools/oecert/host/host.cpp

```diff
@@ -195,37 +214,62 @@ static oe_result_t generate_certificate(
+        "\t%s, %s <format_option>: generate evidence, a report or a "
+        "certificate, where format_option can be one of the following:\n",
+        SHORT_INPUT_PARAM_OPTION_FORMAT,
+        INPUT_PARAM_OPTION_FORMAT);
+    printf(
+        "\t\t%s <private_key> <public_key>: a remote attestation certificate "
+        "in DER format.\n",
         INPUT_PARAM_OPTION_CERT);
     printf(
-        "\t%s <in|out>: use sgx in process or out-of-process quoting.\n",
+        "\t\t%s: a report in OE_FORMAT_UUID_LEGACY_REPORT_REMOTE format.\n",
+        INPUT_PARAM_OPTION_LEGACY_REPORT_REMOTE);
+    printf(
+        "\t\t%s: evidence in OE_FORMAT_UUID_SGX_ECDSA format.\n",
+        INPUT_PARAM_OPTION_SGX_ECDSA);
+    printf(
+        "\t\t%s: evidence in OE_FORMAT_UUID_SGX_EPID_LINKABLE format.\n",
+        INPUT_PARAM_OPTION_SGX_EPID_LINKABLE);
+    printf(
+        "\t\t%s: evidence in OE_FORMAT_UUID_SGX_EPID_UNLINKABLE format.\n",
+        INPUT_PARAM_OPTION_SGX_EPID_UNLINKABLE);
+    printf(
+        "\t%s, %s <in|out>: use SGX in-process or out-of-process quoting.\n",
+        SHORT_INPUT_PARAM_OPTION_QUOTE_PROC,
         INPUT_PARAM_OPTION_QUOTE_PROC);
     printf(
-        "\t%s <output filename> : file for %s, %s, or %s.\n",
-        INPUT_PARAM_OPTION_OUT_FILE,
-        INPUT_PARAM_OPTION_CERT,
-        INPUT_PARAM_OPTION_REPORT,
-        INPUT_PARAM_OPTION_EVIDENCE);
+        "\t%s, %s <filename>: generate an output file for a remote attestation "
+        "certificate, a report or evidence.\n",
+        SHORT_INPUT_PARAM_OPTION_OUT_FILE,
+        INPUT_PARAM_OPTION_OUT_FILE);
     printf(
-        "\t%s : file for endorsements (use with %s or %s).\n",
-        INPUT_PARAM_OPTION_ENDORSEMENTS_FILENAME,
-        INPUT_PARAM_OPTION_REPORT,
-        INPUT_PARAM_OPTION_EVIDENCE);
+        "\t%s, %s <filename>: output a report or evidence, as well as its "
+        "endorsements binary.\n",
+        SHORT_INPUT_PARAM_OPTION_ENDORSEMENTS_FILENAME,
+        INPUT_PARAM_OPTION_ENDORSEMENTS_FILENAME);
     printf(
-        "\t%s : verify the generated %s, %s, or %s\n",
-        INPUT_PARAM_OPTION_VERIFY,
-        INPUT_PARAM_OPTION_CERT,
-        INPUT_PARAM_OPTION_REPORT,
-        INPUT_PARAM_OPTION_EVIDENCE);
+        "\t%s, %s: verify the generated remote attestation certificate, report "
+        "or evidence.\n",
+        SHORT_INPUT_PARAM_OPTION_VERIFY,
+        INPUT_PARAM_OPTION_VERIFY);
     printf(
-        "\t%s <log filename> (default: %s)\n",
+        "\t%s, %s <filename>: generate a log file (default: %s).\n",
+        SHORT_INPUT_PARAM_OPTION_LOG_FILE,
         INPUT_PARAM_OPTION_LOG_FILE,
         DEFAULT_LOG_FILE);
-    printf("\t%s\n", INPUT_PARAM_OPTION_VERBOSE);
+    printf("\t%s: display more information.\n", INPUT_PARAM_OPTION_VERBOSE);
+    printf("Examples:\n");
+    printf("\t1. Show the verification results of evidence in SGX_ECDSA "
+           "format:\n");
+    printf("\t\toecert -f sgx_ecdsa --verify\n");
```

Comment:
Suggest we stick to one type of parameter per sample. I.e. change the first example to
`oecert -f sgx_ecdsa -v`